commit 72a470fd607a40e8f23c1f2407faf69e91350783
Author:     Thomas Deutschmann <whissi@gentoo.org>
AuthorDate: Tue Jun 15 14:11:24 2021 +0200
Commit:     Thomas Deutschmann <whissi@gentoo.org>
CommitDate: Tue Jun 15 14:11:24 2021 +0200

    Revert "meson.build: Change python version check order"
    
    This reverts commit 1b396215d9396680b1b6b5c14c6f57f2226ff0a9.
    
    This is causing a regression for Gentoo, https://bugs.gentoo.org/796143.

diff --git a/meson.build b/meson.build
index 5d507515..1e0e350a 100644
--- a/meson.build
+++ b/meson.build
@@ -252,7 +252,7 @@ libm = cc.find_library('m', required: false)
 libgcab = dependency('libgcab-1.0', version : '>= 1.0', fallback : ['gcab', 'gcab_dep'])
 gcab = find_program('gcab', required : get_option('tests'))
 bashcomp = dependency('bash-completion', required: false)
-python3 = find_program('python3.8', 'python3', 'python3.9')
+python3 = find_program('python3')
 
 if get_option('gnutls')
   gnutls = dependency('gnutls', version : '>= 3.6.0')
